server/graphql/v2/query/ExpenseQuery.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLNonNull, GraphQLInt } from 'graphql';
+
+import { Expense } from '../object/Expense';
+import models from '../../../models';
+
+const ExpenseQuery = {
+  type: Expense,
+  args: {
+    id: {
```

Comment:
Compared to what I said elsewhere about publics ids, I guess we effectively still need to be able to query by id there. Would be great if we didn't have to.

Reply:
Well we have a `idDecode` function. I'll send the coded `id` and decode it before making the db query.

Reply:
After thinking about this one I realized that for the moment we have to use the unencoded version of the `id`. In the frontend we'll be using the expense unencoded id coming from the api v1. I still think it is safe to return the encoded version in the type. Apollo will just register two expenses instead of one.

Reply:
I left a TODO comment there. So that we remember to decode the id after the v1 to v2 migration.